recipes/gnali/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.1.0rc0.dev2" %}
+
+package:
+  name: gnali
+  version: {{ version }}
+
+source:
+  url: https://github.com/phac-nml/gnali/archive/v{{ version }}.tar.gz
+  sha256: c92126ea2bad6335b9f2b3ac6aa9a6a754120c45993709bf4b5df43c484899e0
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - gnali = gnali.gnali:main
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - pip
+  run:
+  - python
+  - pybiomart
```

Comment:
There is no conda package for `pybiomart` - please add that one first in bioconda.

Reply:
Done, thank you for letting me know!